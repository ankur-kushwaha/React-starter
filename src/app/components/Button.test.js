/* global expect*/
import React from "react";
import Button from "./Button";
import renderer from "react-test-renderer";

test("Link changes the class when hovered", () => {
  const component = renderer.create(<Button>Facebook</Button>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
