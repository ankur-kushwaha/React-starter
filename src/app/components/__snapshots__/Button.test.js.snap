// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "white",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }
>
  Facebook
</button>
`;
